### ISSUE TITLE
Tower of Deception: Balance-related problems

### ISSUE THREAD

**Issue by rubyFeedback** [RoxanneSHS/TowerOfDeception#1]
I assume that one goal was to make this challenging. Well, I think right now it is
too challenging or unfair.

(1) That we have to teleport, and then perish means a game-end breaker. I
think you should add more ways out, e. g. pay 50.000 gold or something,
as a dialogue option.

(2) The map in the last battle is too small IMO. My characters can't do 
much. The githyanki just massacre my mages.

(3) The stun is unfair and a game breaker. 3 out of my 6 characters are
stunlocked instantly. So I can't do anything. But not always 3, sometimes
they resist. So it is a bit random this fight.

It seems as if the fight would be easier if I would have taken more 
fighters. I have no strong fighter in this group so I rely a lot on 
spells and survival.

**Comment by RoxanneSHS** [comment]
I fully agree with what you say about the (optional) second component of the mod. It is completely overpowered and unbalanced and designed as a mad old style tactical challenge.
The mod is a very old original and not mine. I just made the update and compatibility for the EE version of the game. The original intention of the Ustraint return was a madness like you experienced.

I can only advise not to install the second option, I never do it myself. However, in maintaining old original mods you must be careful not to destroy the author's intention. So that is what it is. The plus is that it is a fully optional add-on that you can simply skip in installation.

If you have installed part 2 but want to avoid the encounter, enter this into console after you finished the main quest:
C:SetGlobal("VA#StartPart2","Global",9)
